cucumber.yml

```diff
@@ -13,11 +13,20 @@ authenticators: >
   -r cucumber/authenticators/features/support
   -r cucumber/authenticators/features/step_definitions cucumber/authenticators
 
+# NOTE: We have to require the needed files from "api" individually, because
+#       if you mass require the folder it includes "api"s env.rb, which screws
+#       things up because (I think) it sets ENV['CONJUR_ACCOUNT'].  Cucumber
+#       profiles need to be thought through better and refactored most likely.
```

Comment:
This should be filed as an issue in the repo so we don't lose track of it.

Reply:
@micahlee oh, it will :)